### PR TITLE
microbit: use openocd instead of pyocd

### PR DESCRIPTION
**Pull request by aabadie**
This PR provides a way to flash the micro:bit using openocd. The good thing is that now the debugging features of iot-lab can be used with this board.

In the meantime, I dropped the code related to pyocd, since it becomes useless now.

For your information, I ran the integration tests on one CI node in devsaclay, and all passed !

### CODE REVIEW

File: gateway_code/open_nodes/node_microbit.py [hunk 1]
```diff
@@ -52,7 +54,7 @@ class NodeMicrobit(object):
 
     def __init__(self):
         self.serial_redirection = SerialRedirection(self.TTY, self.BAUDRATE)
-        self.ocd = PyOCD(True, self.FLASH_TIMEOUT)
```
Comment (rienafairefr) [inline gateway_code/open_nodes/node_microbit.py:55]: the FLASH_TIMEOUT attribute is ignored so can be removed as well

**Comment by rienafairefr** [comment]
Is this why you added J-link on yocto, is this commit related https://github.com/iot-lab/iot-lab-yocto/commit/3913dfcab478e9f53038353c111d4ef7f8b4fb4c ?

Looks good :+1:

**Review by rienafairefr — APPROVED** [review]

**Comment by aabadie** [comment]
> Is this why you added J-link on yocto

No, I'm working on another open node (NRF52DK). The Yocto addition was required but I have other problem with the serial link.

**Comment by aabadie** [comment]
I removed the FLASH_TIMEOUT member.

**Comment by aabadie** [comment]
@rienafairefr, are you ok to merge this one in the current state ?

**Comment by rienafairefr** [comment]
Good for me 👍

**Comment by aabadie** [comment]
Ok, let's merge this one. Thanks!

**Pull request merged**